tests/data/rtc_gamma_golden.json

```diff
@@ -0,0 +1,95 @@
+{
+  "jobs": [
+    {
+      "name": "{{ name }}",
+      "job_parameters": {
+        "granule": "S1A_IW_SLC__1SSV_20150621T120220_20150621T120232_006471_008934_72D8",
+        "resolution": "10.0",
```

Comment:
I'd prefer to have resolution be an un-quoted number rather than a string, but it clearly doesn't matter in practice